packages/contracts-bedrock/test/L2/CrossL2Inbox.t.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
+import { Reverter, ConfigurableCaller } from "test/mocks/Callers.sol";
+import { ICrossL2Inbox } from "src/L2/ICrossL2Inbox.sol";
+
+// Libraries
+import { SafeCall } from "src/libraries/SafeCall.sol";
+
+import { L1Block } from "src/L2/L1Block.sol";
+
+contract CrossL2InboxTest is CommonTest {
```

Comment:
I recommend not using `CommonTest` for now and just using `Test` and then deploying the contract as part of `setUp` so we don't need to depend on the genesis setup for this PR

Reply:
`CrossL2Inbox.t.sol` is 0.8.15 but CrossL2Inbox is `0.8.24`, so running `ICrossL2Inbox crossL2Inbox = ICrossL2Inbox(new CrossL2Inbox());` in `CrossL2Inbox.t.sol` fails. do you know of any other alternatives?

Reply:
we can make the test `0.8.24`